app/javascript/admin/controllers/config_controller.js

```diff
@@ -93,6 +93,39 @@ export default class ConfigController extends Controller {
     }
   }
 
+  displaySnackbar(message) {
+    return document.dispatchEvent(
+      new CustomEvent('snackbar:add', {
+        detail: { message },
+      }),
+    );
+  }
+
+  async activateLightConfirmation(event) {
+    event.preventDefault();
+    try {
+      const body = new FormData(event.target);
+      const response = await fetch(event.target.action, {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'X-CSRF-Token': document.querySelector("meta[name='csrf-token']")
+            ?.content,
+        },
+        body,
+        credentials: 'same-origin',
+      });
+
+      const outcome = await response.json();
+
+      outcome.message == null
+        ? this.displaySnackbar(outcome.error)
+        : this.displaySnackbar(outcome.message);
```

Comment:
Refactor (non-blocking): Use the [nullish-coalescine operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).

```suggestion
      this.displaySnackbar(outcome.error ?? outcome.message)
```

Reply:
@nickytonline Isn't your suggestion reversed? The first one says display `.error` when `.message == null`, yours is the other way around, isn't it?

Reply:
Updated it. 🙃

Reply:
Today I learned! 😅  Thanks Nick.